blackbox/docs/sql/aggregation.txt

```diff
@@ -330,6 +330,56 @@ Example::
     Due to java double precision arithmetic it is possible that any two executions
     of the aggregation function on the same data produce slightly differing results.
 
+percentile
+==========
+
+The ``percentile`` aggregation function computes a `Percentile`_ over numeric
+non-null values in a column.
+
+Percentiles show the point at which a certain percentage of observed values occur. 
+For example, the 98th percentile is the value which is greater than 98% of the observed values.
+
+The function expects a single fraction or an array of fractions and a column name.
+If the value at the specified column is ``null`` the row is ignored.
+Fractions must be double precision values between 0 and 1.
+When supplied a single fraction, the function will return a single value corresponding
+to the percentile of the specified fraction::
```

Comment:
i think this is wrong, the return type should be the input type. so for example a 0.5 percentile should be the median